src/sql/workbench/contrib/assessment/test/common/vsServicesStubs.ts

```diff
@@ -0,0 +1,205 @@
+	resolveAll(toResolve: { resource: URI, options?: files.IResolveFileOptions }[]): Promise<files.IResolveFileResult[]> {
+		throw new Error('Method not implemented.');
+	}
+	exists(resource: URI): Promise<boolean> {
+		throw new Error('Method not implemented.');
+	}
+	readFile(resource: URI, options?: files.IReadFileOptions): Promise<files.IFileContent> {
+		throw new Error('Method not implemented.');
+	}
+	readFileStream(resource: URI, options?: files.IReadFileOptions): Promise<files.IFileStreamContent> {
+		throw new Error('Method not implemented.');
+	}
+	writeFile(resource: URI, bufferOrReadableOrStream: VSBuffer | VSBufferReadable | VSBufferReadableStream, options?: files.IWriteFileOptions): Promise<files.IFileStatWithMetadata> {
+		throw new Error('Method not implemented.');
+	}
+	move(source: URI, target: URI, overwrite?: boolean): Promise<files.IFileStatWithMetadata> {
+		throw new Error('Method not implemented.');
+	}
+	async canMove(source: URI, target: URI, overwrite?: boolean | undefined): Promise<Error | true> { return true; }
+	copy(source: URI, target: URI, overwrite?: boolean): Promise<files.IFileStatWithMetadata> {
+		throw new Error('Method not implemented.');
+	}
+	async canCopy(source: URI, target: URI, overwrite?: boolean | undefined): Promise<Error | true> { return true; }
+	async canCreateFile(source: URI, options?: files.ICreateFileOptions): Promise<Error | true> { return true; }
+	createFile(resource: URI, bufferOrReadableOrStream?: VSBuffer | VSBufferReadable | VSBufferReadableStream, options?: files.ICreateFileOptions): Promise<files.IFileStatWithMetadata> {
+		return Promise.resolve({
+			ctime: 0,
+			etag: '',
+			isDirectory: false,
+			isFile: false,
+			isSymbolicLink: false,
+			mtime: 0,
+			name: '',
+			resource: resource,
+			size: 0
+		});
+	}
+	createFolder(resource: URI): Promise<files.IFileStatWithMetadata> {
+		throw new Error('Method not implemented.');
+	}
+	del(_resource: URI, _options?: { useTrash?: boolean, recursive?: boolean }): Promise<void> { return Promise.resolve(); }
+	canDelete(resource: URI, options?: { useTrash?: boolean | undefined; recursive?: boolean | undefined; } | undefined): Promise<Error | true> {
+		return Promise.resolve(true);
+	}
+	readonly watches: URI[] = [];
+	watch(_resource: URI): IDisposable {
+		this.watches.push(_resource);
+
+		return null;
+	}
+	dispose(): void { }
+
+}
+
+export class DialogServiceStub implements IDialogService {
+	_serviceBrand: undefined;
+	confirm(confirmation: IConfirmation): Promise<IConfirmationResult> {
+		throw new Error('Method not implemented.');
+	}
+	show(severity: Severity, message: string, buttons: string[], options?: IDialogOptions): Promise<IShowResult> {
+		throw new Error('Method not implemented.');
+	}
+	about(): Promise<void> {
+		throw new Error('Method not implemented.');
+	}
+}
+
+export class ClipboardServiceStub implements IClipboardService {
```

Comment:
Looks like we don't have this one yet - I'd suggest adding it to https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/test/workbenchTestServices.ts#L1 instead